lib/matplotlib/tests/test_polar.py

```diff
@@ -481,3 +481,16 @@ def test_polar_neg_theta_lims():
     ax.set_thetalim(-np.pi, np.pi)
     labels = [l.get_text() for l in ax.xaxis.get_ticklabels()]
     assert labels == ['-180°', '-135°', '-90°', '-45°', '0°', '45°', '90°', '135°']
+
+
+@image_comparison(baseline_images=['polar_errorbar'], remove_text=False,
```

Comment:
```suggestion
@image_comparison(baseline_images=['polar_errorbar'], remove_text=True,
```
Please remove text if it is not essential for the test. Font rendering is complex and updates of the renderer can lead to minor visual changes, which could trigger the image comparison.